src/OrchardCore.Modules/OrchardCore.Contents/Scripting/ContentMethodsProvider.cs

```diff
@@ -51,19 +48,26 @@ public ContentMethodsProvider()
                 Method = serviceProvider => (Action<ContentItem, object>)((contentItem, properties) =>
                 {
                     var contentManager = serviceProvider.GetRequiredService<IContentManager>();
-                    var props = JObject.FromObject(properties);
-                    var content = (JObject)contentItem.Content;
-
-                    content.Merge(props, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
-                    contentItem.Apply(props.ToObject<ContentItem>());
+                    contentItem.Merge(properties, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
                     contentManager.UpdateAsync(contentItem).GetAwaiter().GetResult();
                 })
             };
+
+            _deleteContentItemMethod = new GlobalMethod
+            {
+                Name = "deleteContentItem",
+                Method = serviceProvider => (Action<ContentItem, object>)((contentItem, properties) =>
+                {
+                    var contentManager = serviceProvider.GetRequiredService<IContentManager>();
+                    contentManager.RemoveAsync(contentItem).GetAwaiter().GetResult();
```

Comment:
I need to test if calling a RemoveAsync for a ContentItem that does not exist would fail.